kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2005,7 +2005,7 @@ object KyuubiConf {
       " <li>HIVE_SQL: specify this engine type will launch a Hive engine which can provide" +
       " all the capacity of the Hive Server2.</li>" +
       " <li>JDBC: specify this engine type will launch a JDBC engine which can provide" +
-      " a MySQL protocol connector, for now we only support Doris dialect.</li>" +
+      " a MySQL protocol connector, for now we support Doris dialect and Phoenix dialect.</li>" +
```

Comment:
JDBC: specify this engine type will launch a JDBC engine which can forward queries to the database system through the certain JDBC driver, for now, it supports Doris and Phoenix.

Reply:
thanks for the review，updated。